lib/utils.go

```diff
@@ -0,0 +1,12 @@
+package lib
```

Comment:
Would it make sense to name it according to the file name?
```suggestion
package utils
```

Reply:
Then we would move this file into a new package. i don't think this would be necessary at this moment.
You might be mixing something up here

Reply:
Okay, I'm not very familiar with this bits in Go. Just thought `utils` would better match helper functions.